bootable-jar/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>testable-examples-pom</artifactId>
+        <version>4.1.2.Final-SNAPSHOT</version>
```

Comment:
Hmmmm,  version is "4.1.2.Final-SNAPSHOT".   Shouldn't it be either a  Final or a SNAMPSHOT?